lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -130,27 +131,29 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     EngineFlutterDisplay.instance,
   ];
 
+  late FlutterViewManager viewManager;
```

Comment:
Yeah, you're probably right. I just like to put things in the constructor because then it's very easy to go to the method that cleans up and see what you need to do with each line that is created there :P (I'll move it)

Reply:
(Removed viewId handling from the view manager)